addons-dev/aldryn-addons/aldryn_addons/utils.py

```diff
@@ -0,0 +1,37 @@
+# -*- coding: utf-8 -*-
+import json
+import os
+
+
+def boolean_ish(value):
+    if isinstance(value, basestring):
+        value = value.lower()
+    if value in [True, 'true', '1', 'on', 'yes']:
+        return True
+    elif value in [False, 'false', '0', 'off', 'no']:
+        return False
+    return bool(value)
+
+
+def json_from_file(path):
+    try:
+        with open(path) as fobj:
+            return json.load(fobj)
+    except ValueError as e:
+        raise ValueError('{} ({})'.format(e, path))
+
+
+def mkdirs(path):
+    try:
+        os.makedirs(path)
+    except:
```

Comment:
Hm.... whenever I see `except` without exception type I'm suspicious. I'm kinda picky, but ... I'd put this `os.error` here (https://docs.python.org/2/library/os.html)